### PR TITLE
ci: disable cluster-test LBT

### PR DESCRIPTION
**Pull request by bmwill**

### CODE REVIEW

File: .github/workflows/ci-test.yml [hunk 1]
```diff
@@ -1266,51 +1081,3 @@ jobs:
         uses: ./.github/actions/early-terminator
         with:
           github-token: ${{secrets.GITHUB_TOKEN}}
-
```
Comment (zihaoccc) [inline .github/workflows/ci-test.yml:1269]: we should also remove the step `- name: Post test results on PR` from line 1068 which is used to post temp test result for comparison only

Comment (bmwill) [reply .github/workflows/ci-test.yml:1269]: Do you mean line 1253? we still want forge to post results to the PR with the perf numbers

**Comment by bmwill** [comment]
@zihaoccc Do you know what happened on this run of forge?

File: .github/workflows/ci-test.yml (hunk 1 above)
Comment (zihaoccc) [reply .github/workflows/ci-test.yml:1269]: yea, sorry for a little bit confusion, there is an another step above also named Post test results on PR, that is the actual step to post perf number

Comment (zihaoccc) [reply .github/workflows/ci-test.yml:1269]: As a temporary step I assume it will be removed shortly, I might forgot to change the name 😄

**Comment by zihaoccc** [comment]
> @zihaoccc Do you know what happened on this run of forge?

first time saw it, seems connection broken
```
1: error sending request for url (http://10.100.219.64/v1): connection closed before message completed
    2: connection closed before message completed
```
let's retry

File: .github/workflows/ci-test.yml (hunk 1 above)
Comment (bmwill) [reply .github/workflows/ci-test.yml:1269]: I think i removed the correct one now

**Review by gregnazario — APPROVED** [review]

**Review by rustielin — APPROVED** [review]

**Review by zihaoccc — DISMISSED** [review]

**Pull request closed without merging**